.github/workflows/repo.hex.pm.yml

```diff
@@ -20,10 +20,12 @@ jobs:
       matrix:
         include:
           - otp: 24
-            otp_version: 24.3
+            otp_version: '24.3'
             upload_generic_version: upload_generic_version
           - otp: 25
-            otp_version: 25.0
+            otp_version: '25.3'
```

Comment:
Question: should I add `upload_generic_version: upload_generic_version` here too?

Reply:
No, it is always on the lowest supported version. :)